src/Compiler/perf/Microsoft.AspNetCore.Razor.Microbenchmarks.Generator/ParamsAllExceptDebug.cs

```diff
@@ -0,0 +1,24 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using BenchmarkDotNet.Attributes;
+
+namespace Microsoft.AspNetCore.Razor.Microbenchmarks.Generator;
+internal class ParamsAllExceptDebugAttribute
+#if DEBUG
+    : ParamsAttribute
+#else
+    : ParamsAllValuesAttribute
+#endif
+
+{
+    internal ParamsAllExceptDebugAttribute(params object[] args)
```

Comment:
Nit: a better name would be `ParamsAllUnlessDebugAttribute`.

Reply:
Also, doesn't BenchmarkDotNet complain when run in debug mode?

Reply:
Yeah, it's only for locally debugging the benchmarks.